app/producers/notification.rb

```diff
@@ -26,7 +26,10 @@ def self.deliver(event_type:, account:, host:, policy:, **extra)
         metadata: {},
         payload: payload
       }],
-      context: payload
+      context: {
+        host: host.id,
```

Comment:
Can we follow a convention similar to Advisor? Here's an example:
```
'context': {
  'display_name': 'rhel8desktop',
  'host_url': 'https://console.redhat.com/insights/inventory/10000000-1000-4000-8000-100000000000',
  'hostname': 'rhel8desktop',
  'inventory_id': '10000000-1000-4000-8000-100000000000',
  'rhel_version': '8.3',
  'tags': [{'key': 'Team', namespace': 'insights-client',  'PMs'}]
}
```

Reply:
It is not a convention yet, we're working on it... From what I've heard, the `context` can have anything that is common within the `events` array. Hence the `policy` id can be included as well.

I've added Advisor's example, as it includes more verbose host identification, including display name and RHEL version. There is no standard on the identification yet, unfortunately.

Reply:
Sorry to say that, but this is a mess then :see_no_evil: :confused: the policy and host are equally part of the context and with this logic, however, if we aggregate, the policy will have a higher priority. We surely can have a single host assigned to multiple policies, but the opposite (multiple hosts under a policy) is the way people use compliance. So I am not entirely sure if a host even should be part of our context :thinking:

Reply:
I should say that the concern is mainly on the identification of a system/host. If it would be an event with a single policy having multiple systems reporting, then the policy can be in context and individual systems in the events array.